schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
+    id INT,
```

Comment:
[OPTIONAL]
- You can make the id column not null and autoincremented. 
` id int NOT NULL IDENTITY(1,1)`
- You can also mark this column as the primary key for example:
`PRIMARY KEY (id)`

Reply:
Thank you.
I now have the images added to my README.md